clang/test/CodeGenHLSL/builtins/ddy-coarse.hlsl

```diff
@@ -0,0 +1,61 @@
+// RUN: %clang_cc1 -finclude-default-header  -x hlsl  -triple dxil-pc-shadermodel6.3-library %s \
+// RUN:  -emit-llvm -disable-llvm-passes -fnative-half-type -o - | \
+// RUN:  FileCheck %s --check-prefixes=CHECK
```

Comment:
like the ddx test case add a spirv  target triple run line.

Reply:
Done